packages/component-library/stories/HexOverlay.story.js

```diff
@@ -85,17 +92,55 @@ const demoMap = () => {
       <HexOverlay
         data={data.features}
         opacity={opacity}
+        coverage={coverage}
         radius={radius}
         elevation={elevation}
-        extruded={extruded}
         colorRange={colorRange}
         lightSettings={lightSettings}
+        filled={filled}
+        wireframe={wireframe}
       />
     </BaseMap>
   );
 };
 
+// with tooltip version
+const tooltipMap = () => {
+  const coverage = number('Coverage:', 0.8, coverageOptions);
+  const opacity = number('Opacity:', 0.8, opacityOptions);
+  const radius = number('Radius', 500, radiusOptions);
+  const elevation = number('Elevation:', 10, elevationOptions);
+  const extruded = boolean('Extruded:', true);
+  const filled = boolean('Filled:', true);
+  const wireframe = boolean('Wireframe:', true);
+
+  const mapboxStyle = selectV2('Mapbox Style', optionsStyle, optionsStyle['Label Maker']);
+  return (
+    <BaseMap
+      mapboxToken={mapboxToken}
+      mapboxStyle={mapboxStyle}
+    >
+      <HexOverlay
+        data={data.features}
+        opacity={opacity}
+        coverage={coverage}
+        radius={radius}
+        elevation={elevation}
+        colorRange={colorRange}
+        lightSettings={lightSettings}
+        filled={filled}
+        wireframe={wireframe}
+      >
+        <MapTooltip isHex={true} />
```

Comment:
Requiring the user of HexOverlay to know that the tooltip needs this `isHex` flag to not throw errors is also a really bad developer experience.